File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/SealOfClaris.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.Item;

/* JADX INFO: loaded from: classes3.dex */
public class SealOfClaris extends Item {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.item_seal_of_claris_name;
        this.idDescription = R.string.item_seal_of_claris_description;
        this.idImage = R.drawable.seal_of_claris;
        this.source.add(Integer.valueOf(R.string.raid_name_the_cultist_rebels));
        this.price = 6500L;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/SealOfClaris.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.Item

class SealOfClaris : Item() {
    override fun configureProperties() {
        idName = R.string.item_seal_of_claris_name
        idDescription = R.string.item_seal_of_claris_description
        idImage = R.drawable.seal_of_claris
        source.add(R.string.raid_name_the_cultist_rebels)
        price = 6500L
    }
}
